src/operations/expansions/expandGeom2.js

```diff
@@ -0,0 +1,20 @@
+const { geom2 } = require('../../geometry')
+
+const offsetGeom2 = require('./offsetGeom2')
+
+/**
+ * Expand the given geometry (geom2) using the given options (if any).
+ * @param {Object} options - options for expand
+ * @param {Number} [options.delta=1] - delta (+/-) of expansion
+ * @param {Integer} [options.segments=0] - number of segments when creating rounded corners, or zero for chamfer
```

Comment:
hmm I both like this (using segments === 0 for chamfer) , if we had a non polygon based system, would you say that passing Infinity (an actual js thing) to this value would generate something with maximum resolution/ no polygons?

Reply:
Yes. I think you are correct.
Maybe would should say 'chamfer' instead of segments? Where 'chamfer':
- 0 : close corners
- 1 : close corners at intersect
- N : close corners with rounding (of course, we use count of segments now)